nomad/structs/structs.go

```diff
@@ -89,6 +89,17 @@ const (
 	GetterModeDir  = "dir"
 )
 
+// Context is a type which is searchable via a unique identifier.
```

Comment:
Comment doesn't make much sense.

`Context defines the scope in which a search for Nomad object operates`